docs/cli/vs.rst

```diff
@@ -291,6 +291,10 @@ If no timezone is specified, IMS local time (CST) will be assumed, which might n
    :prog: virtual notification-delete
    :show-nested:
 
+.. click:: SoftLayer.CLI.virt.host_list:cli
+    :prog: virtual host-list
```

Comment:
```suggestion
.. click:: SoftLayer.CLI.virt.host_list:cli
    This command is an alias for `slcli dedicatedhost list`
    :prog: virtual host-list
```

Reply:
Done

Reply:
@allmightyspiff Also was added 
```
.. click:: SoftLayer.CLI.dedicatedhost.list:cli
    :prog: virtual host-list
    :show-nested:
```
in `docs/cli/dedicated.rst` to resolve an error generated when running  `python docCheck.py`
```
===> virtual:host-list SoftLayer.CLI.dedicatedhost.list:cli IS UNDOCUMENTED <===
```